### PR TITLE
retire DeviceNativeBoundsSize

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

retire outdated `DeviceNativeBoundsSize` helpers

### CODE REVIEW

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -727,12 +727,11 @@
 		A949418923E09489001B0373 /* SplitViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = A949418823E09489001B0373 /* SplitViewController.swift */; };
 		A949418B23E09AFB001B0373 /* ConversationContentViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = A949418A23E09AFB001B0373 /* ConversationContentViewController.swift */; };
 		A949418D23E180FF001B0373 /* Bundle+config.swift in Sources */ = {isa = PBXBuildFile; fileRef = A949418C23E180FF001B0373 /* Bundle+config.swift */; };
-		A9536FD523ACE12900CFD528 /* SwiftMockUser.swift in Sources */ = {isa = PBXBuildFile; fileRef = A9536FD423ACE12900CFD528 /* SwiftMockUser.swift */; };
```
Comment (johnxnguyen) [inline Wire-iOS.xcodeproj/project.pbxproj]: Unrelated?

Comment (billypchan) [reply Wire-iOS.xcodeproj/project.pbxproj]: It is leave-over from pervious PR, but XCode does the clean up automatically.
